addons/l10n_latam_check/models/account_payment.py

```diff
@@ -219,10 +219,10 @@ def _compute_l10n_latam_check_warning_msg(self):
                 same_checks = self.env['l10n_latam.check']
                 for check in rec.l10n_latam_new_check_ids.filtered(
                         lambda x: x.name and x.payment_method_line_id.code == 'new_third_party_checks' and
-                        x.bank_id and x.issuer_vat):
+                        x.partner_bank_id and x.issuer_vat):
                     same_checks += same_checks.search([
                         ('company_id', '=', rec.company_id.id),
-                        ('bank_id', '=', check.bank_id.id),
+                        ('partner_bank_id', '=', check.partner_bank_id.id),
```

Comment:
this code is dodgy... search in a loop etc...
I wonder if the bank check still makes sense
I guess we can do it this way and see if it become an issue and fix the code then